charts/pulsar/Chart.yaml

```diff
@@ -18,7 +18,7 @@
 #
 
 apiVersion: v1
-appVersion: "2.7.2"
+appVersion: "2.7.4"
 description: Apache Pulsar Helm chart for Kubernetes
 name: pulsar
 version: 2.7.6
```

Comment:
@Technoboy- you need to bump the chart version to 2.7.7

Reply:
Did you mean :  version: 2.7.6 -> version: 2.7.7?

Reply:
Ok, thanks @lhotari